src/constructors/constructWithOptions.js

```diff
@@ -1,4 +1,5 @@
 // @flow
+import * as ReactIs from 'react-is'
```

Comment:
Can you destructure this import please to make it more explicit? We really only need to import `isValidElementType`